torch/_dynamo/variables/tensor.py

```diff
@@ -158,13 +158,16 @@ def specialize(value: torch.Tensor):
                 [int(s) if is_symbolic(s) else s for s in value.size()]
             )
             props["stride"] = tuple(value.stride())
-            props["is_contiguous"] = tuple(
-                [
-                    x
-                    for x in torch._prims_common._memory_formats
-                    if value.is_contiguous(memory_format=x)
-                ]
-            )
+            if torch._C._functorch.is_batchedtensor(value):
```

Comment:
Can you add a note saying that batchedtensor doesn't support all contiguity patterns, so we don't cache is_contiguous on it here?